elk/training/train.py

```diff
@@ -83,12 +83,9 @@ def apply_to_layer(
             reporter = CcsReporter(self.net, d, device=device, num_variants=v)
             train_loss = reporter.fit(first_train_h)
 
-            if not self.net.norm == "burns":
-                (_, v, k, _) = first_train_h.shape
-                reporter.platt_scale(
-                    to_one_hot(repeat(train_gt, "n -> (n v)", v=v), k).flatten(),
-                    rearrange(first_train_h, "n v k d -> (n v k) d"),
-                )
+            labels = to_one_hot(train_gt, k)
+            labels = repeat(train_gt, "n -> n v k", v=v, k=k)
```

Comment:
Why do we repeat the labels along the choices dimension? I think this means all dimensions of the one-hot prediction are expected to have the same value.

CCS only supports k=2, so we can just deal with that case after after asserting it true.

I think this line should be `labels = to_one_hot(repeat(train_gt, "n -> (n v)", v=v), 2).flatten()`

Reply:
`labels = to_one_hot(repeat(train_gt, "n -> (n v)", v=v), 2).flatten()`
That wouldn't work, since we dimensions have to be the same for the labels and first_train_h, which is a 3D tensor in the case of CCS